mgmt/api/include/mgmtapi.h

```diff
@@ -488,7 +488,7 @@ tsapi TSMgmtError TSReadFromUrl(char *url, char **header, int *headerSize, char
  */
 tsapi TSMgmtError TSReadFromUrlEx(const char *url, char **header, int *headerSize, char **body, int *bodySize, int timeout);
 tsapi TSMgmtError TSHostStatusSetUp(const char *name);
-tsapi TSMgmtError TSHostStatusSetDown(const char *name);
+tsapi TSMgmtError TSHostStatusSetDown(const int down_time, const char *name);
```

Comment:
I think this should be TSHostStatusSetDown(const char*name, int down_time);   I also don't think it has to be "const int" on down_time.